yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this file is not needed because since we do not have any dependencies in the root of monorepo

Reply:
good catch, removed